tests/test_assets.py

```diff
@@ -450,16 +457,41 @@ def test_lookup_symbol(self):
                 self.assertEqual(result.symbol, 'EXISTING')
                 self.assertEqual(result.sid, i)
 
-    @parameterized.expand(
-        build_lookup_generic_cases()
-    )
-    def test_lookup_generic(self, finder, symbols, reference_date, expected):
+    def test_lookup_symbol_from_multiple_valid(self):
```

Comment:
I added this test because of a mistake I caught when implementing the cached functionality for lookup_symbol. I had initially forgotten to sort multiple valid equities found for a symbol by start date/end date. The tests had still passed, even though a different sid was being returned than in the non-caching version.